File: app/src/main/java/com/zhaoweihao/mrtranslator/Util/HttpUtil.java
package com.zhaoweihao.mrtranslator.Util;

import okhttp3.OkHttpClient;
import okhttp3.Request;

/**
 * Created by Administrator on 2017/4/13.
 */

public class HttpUtil {
    public static void sendOkHttpRequest(String address,okhttp3.Callback callback){
        OkHttpClient client=new OkHttpClient();
        Request request=new Request.Builder()
                .url(address)
                .build();
        client.newCall(request).enqueue(callback);
    }
}

File: app/src/main/java/com/zhaoweihao/mrtranslator/utils/HttpUtil.kt
package com.zhaoweihao.mrtranslator.utils

import okhttp3.Callback
import okhttp3.OkHttpClient
import okhttp3.Request

/**
 * Created by Zhaoweihao on 2017/4/13.
 */
object HttpUtil {
    fun sendOkHttpRequest(address: String?, callback: Callback?) {
        val client = OkHttpClient()
        val request = Request.Builder()
                .url(address)
                .build()
        client.newCall(request).enqueue(callback)
    }
}